fs/localfs/local_fs.go

```diff
@@ -356,6 +356,7 @@ func Directory(path string) (fs.Directory, error) {
 func entryFromDirEntry(fi os.FileInfo, prefix string) fs.Entry {
 	isplaceholder := strings.HasSuffix(fi.Name(), ShallowEntrySuffix)
 	maskedmode := fi.Mode() & os.ModeType
+	readNamedEnv := os.Getenv("KOPIA_SNAPSHOT_NAMED_PIPES")
```

Comment:
this function is called in a tight loop, reading from environment has non-trivial cost and should be avoided in such a place.

Reply:
Thanks for reviewing :)

I‘ll update the PR to cache the value.

Reply:
I moved the variable outside the function. This should fix the performance issues.